T6335: Add/Update EVPN op commands

l0crian1 opened the pull request:
<!-- All PR should follow this template to allow a clean and transparent review -->
<!-- Text placed between these delimiters is considered a comment and is not rendered -->

## Change Summary
<!--- Provide a general summary of your changes in the Title above -->
Added the following commands:
show evpn
show evpn es
show evpn es <es-id>
show evpn es detail
show evpn es-evi
show evpn es-evi detail
show evpn es-evi vni <num>
show evpn vni
show evpn vni detail
show evpn vni <num>

Updated the following commands:
show evpn access-vlan
show evpn arp-cache
show evpn mac
show evpn next-hops
show evpn rmac
## Types of changes
<!---
What types of changes does your code introduce? Put an 'x' in all the boxes that apply.
NOTE: Markdown requires no leading or trailing whitespace inside the [ ] for checking
the box, please use [x]
-->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)
<!-- optional: Link to related other tasks on Phabricator. -->
<!-- * https://vyos.dev/Txxxx -->
https://vyos.dev/T6335

## Related PR(s)
<!-- Link here any PRs in other repositories that are required by this PR -->

## Component(s) name
<!-- A rather incomplete list of components: ethernet, wireguard, bgp, mpls, ldp, l2tp, dhcp ... -->
evpn, op-mode
## Proposed changes
<!--- Describe your changes in detail -->
Additional EVPN op commands that are available in FRR have been exposed to VyOS. Additionally some existing commands were updated with exposure to the API as well as completion helpers.
## How to test
<!---
Please describe in detail how you tested your changes. Include details of your testing
environment, and the tests you ran. When pasting configs, logs, shell output, backtraces,
and other large chunks of text, surround this text with triple backtics
```
like this
```
-->
Configure EVPN and run show commands:
Examples:
```
vyos@vyos:~$ show evpn es
Type: B bypass, L local, R remote, N non-DF
ESI                            Type ES-IF                 VTEPs
03:aa:bb:cc:dd:ee:f0:00:00:64  LN   bond0
03:aa:bb:cc:dd:ee:f2:00:00:65  LN   bond1
```

```
vyos@vyos:~$ show evpn vni
VNI        Type VxLAN IF              # MACs   # ARPs   # Remote VTEPs  Tenant VRF
102        L2   vxlan2                0        0        0               default
100        L2   vxlan0                0        0        0               default
105        L2   vxlan5                0        0        0               default
103        L2   vxlan3                0        0        0               default
109        L2   vxlan9                0        0        0               default
110        L2   vxlan10               0        0        0               default
108        L2   vxlan8                0        0        0               default
101        L2   vxlan1                0        0        0               default
107        L2   vxlan7                0        0        0               default
106        L2   vxlan6                0        0        0               default
104        L2   vxlan4                0        0        0               default
```

```
vyos@vyos:~$ show evpn vni 100
VNI: 100
 Type: L2
 Vlan: 1
 Bridge: br0
 Tenant VRF: default
 VxLAN interface: vxlan0
 VxLAN ifIndex: 19
 SVI interface: br0
 SVI ifIndex: 6
 Local VTEP IP: 10.1.2.2
 Mcast group: 0.0.0.0
 No remote VTEPs known for this VNI
 Number of MACs (local and remote) known for this VNI: 0
 Number of ARPs (IPv4 and IPv6, local and remote) known for this VNI: 0
 Advertise-gw-macip: No
 Advertise-svi-macip: No

```
## Smoketest result
<!-- Provide the output of the smoketest
```
$ /usr/libexec/vyos/tests/smoke/cli/test_xxx_feature.py
test_01_simple_options (__main__.TestFeature.test_01_simple_options) ... ok
```
-->

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
<!--- The entire development process is outlined here: https://docs.vyos.io/en/latest/contributing/development.html -->
- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [ ] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

l0crian1 committed:
T6335: Add/Update EVPN op commands

Added the following commands:
show evpn
show evpn es
show evpn es <es-id>
show evpn es detail
show evpn es-evi
show evpn es-evi detail
show evpn es-evi vni <num>
show evpn vni
show evpn vni detail
show evpn vni <num>

Updated the following commands:
show evpn access-vlan
show evpn arp-cache
show evpn mac
show evpn next-hops
show evpn rmac

c-po reviewed (changes requested):
Python completion helpers tend to be super slow as the interpreter needs to be started up. It would be better to have the completion helpers available in bash.

l0crian1 committed:
T6335: Add/Update EVPN op commands

Converted completion helpers from python to bash for performance

Previous commit:
Added the following commands:
show evpn
show evpn es
show evpn es <es-id>
show evpn es detail
show evpn es-evi
show evpn es-evi detail
show evpn es-evi vni <num>
show evpn vni
show evpn vni detail
show evpn vni <num>

Updated the following commands:
show evpn access-vlan
show evpn arp-cache
show evpn mac
show evpn next-hops
show evpn rmac

l0crian1 commented:
> Python completion helpers tend to be super slow as the interpreter needs to be started up. It would be better to have the completion helpers available in bash.

Thanks, I was actually wondering why tab completion on partials felt a little sluggish. Done!

c-po commented:
@MergifyIo backport sagitta
